### DISCUSSION TITLE
feat: improvement in tag page

### DISCUSSION THREAD

**Discussion by kranurag78** [hugo-sid/hugo-blog-awesome#14] [Feature requests]
As of now, our tag page looks something like this. 

![image](https://user-images.githubusercontent.com/84301342/224725186-96cc624c-1856-4ad2-8f78-682de0a493e2.png)

I think it will be great, if we cover the number of articles corresponding to a tag and we should remove the date from the tags page as well. On the tags page, we can list tags in this manner `<tag-name>(number)`

**Comment by hugo-sid** [comment]
Hi Anurag, thanks for pointing out the issues on tag page.

> I think it will be great, if we cover the number of articles corresponding to a tag and we should remove the date from the tags page as well. On the tags page, we can list tags in this manner `<tag-name>(number)`.

Nice idea indeed ! I will try to implement this, once I get some time.

**Comment by Vctrsnts** [comment]
Thanks a lot for this work, but is have 2 o 3 tags? how put?
Because if have one is this:
```
tags =
  - tag 1
```

But with 2 o 3?

Thanks a lot

**Reply by Vctrsnts** [reply]
The problem is my. Sorry.

Is this:
```
tags:
 - tag 1
 - tag 2
 - etc...
```

Sorry and thanks for this great work
